scanner/src/main/kotlin/PostgresStorage.kt

```diff
@@ -100,10 +98,12 @@ class PostgresStorage(val connection: Connection) : ScanResultsStorage() {
     }
 
     override fun readFromStorage(id: Identifier): ScanResultContainer {
+        log.info { "Reading scan results for ${id.toCoordinates()} from storage." }
+
         val query = "SELECT scan_result FROM $schema.$table WHERE identifier = ?"
 
         val statement = connection.prepareStatement(query)
-        statement.setString(1, id.toString())
+        statement.setString(1, id.toCoordinates())
```

Comment:
Should this be squash to where the `toString()` call was introduced?

Reply:
Done.